File: market-mapper/src/main/java/by/market/mapper/DataTypeMapper.java
package by.market.mapper;

import by.market.domain.system.DataType;
import by.market.dto.system.DataTypeDTO;
import org.mapstruct.Mapper;

@Mapper(config = MapperConfiguration.class)
public interface DataTypeMapper extends MapstructMapper<DataTypeDTO, DataType> {
}

File: market-mapper/src/main/java/by/market/mapper/system/DataTypeMapper.kt
package by.market.mapper.domain_dto_mapper.system

import by.market.domain.system.DataType
import by.market.dto.system.DataTypeDTO
import by.market.mapper.IMapstructMapper
import by.market.mapper.MapperConfig
import org.mapstruct.Mapper

@Mapper(config = MapperConfig::class)
interface DataTypeMapper : IMapstructMapper<DataTypeDTO, DataType>
